src/cms/templates/linkcheck/links_by_filter.html

```diff
@@ -0,0 +1,59 @@
+{% extends '_base.html' %}
+{% load i18n %}
+
+{% block content %}
+  <h1 class="heading">
+    {% if link_filter == 'valid' %}
+      {% trans 'Valid links' %} ({{ number_valid }})
+    {% elif link_filter == 'invalid' %}
+      {% trans 'Invalid links' %} ({{ number_invalid }})
+    {% elif link_filter == 'unchecked' %}
+      {% trans 'Unchecked links' %} ({{ number_unchecked }})
+    {% else %}
+      {% trans 'Ignored links' %} ({{ number_ignored }})
+    {% endif %}
+  </h1>
+  <a href="{% if link_filter == 'valid' %}#{% else %}{% url 'linkcheck' region_slug=region_slug link_filter='valid' %}{% endif %}" 
```

Comment:
```suggestion
  <a href="{% if link_filter == 'valid' %}#{% else %}{% url 'linkcheck' region_slug=region.slug link_filter='valid' %}{% endif %}" 
```
No reason for the additional context variable `region_slug`, since you already have the current region available through the [region_slug_processor](https://github.com/Integreat/integreat-cms/blob/5d01d756e2fcf1719aba60611f5699c5655485f7/src/backend/context_processors.py#L7)